pandas/io/tests/test_json/test_pandas.py

```diff
@@ -815,11 +815,39 @@ def test_mixed_timedelta_datetime(self):
 
     def test_default_handler(self):
         value = object()
-        frame = DataFrame({'a': ['a', value]})
-        expected = frame.applymap(str)
+        frame = DataFrame({'a': [7, value]})
+        expected = DataFrame({'a': [7, str(value)]})
         result = pd.read_json(frame.to_json(default_handler=str))
         assert_frame_equal(expected, result, check_index_type=False)
 
+    def test_default_handler_indirect(self):
```

Comment:
It's already covered [here](http://pandas-docs.github.io/pandas-docs-travis/io.html#fallback-behavior) but I brought it up to date.